waterlog/src/reducers/index.js

```diff
@@ -3,10 +3,15 @@ import sensorReducer from './sensorReducer';
 import segmentsReducer from './segmentsReducer';
 import segmentEventReducer from './segmentEventsReducer';
 import segmentLeaksReducer from './segmentLeaksReducer';
```

Comment:
Consider to make less reducers and combined them into one if they fit the same kind of responsibility